src/Access/AccessRights.cpp

```diff
@@ -205,7 +214,7 @@ namespace
         switch (level)
         {
             case GLOBAL_LEVEL: return AccessFlags::allFlagsGrantableOnGlobalLevel();
-            case DATABASE_LEVEL: return AccessFlags::allFlagsGrantableOnDatabaseLevel();
+            case DATABASE_LEVEL: return AccessFlags::allFlagsGrantableOnDatabaseLevel() | AccessFlags::allFlagsGrantableOnGlobalWithParameterLevel();
```

Comment:
Let's add

```
GLOBAL_WITH_PARAMETER_LEVEL = DATABASE_LEVEL
```

with a comment to the Level enum above?

Reply:
done